packages/build/src/plugins_core/functions/index.js

```diff
@@ -100,10 +119,18 @@ const coreCommand = async function ({
 
   await validateFunctionsSrc({ functionsSrc, relativeFunctionsSrc })
 
-  const functions = await getFunctionPaths(functionsSrc)
-  logFunctionsToBundle(logs, functions, relativeFunctionsSrc)
+  const userFunctions = await getFunctionPaths(functionsSrc)
+  const internalFunctions = await listInternalFunctions(internalFunctionsSrc)
```

Comment:
Should we use `Promise.all()` to parallelize both async calls?

Reply:
Done in https://github.com/netlify/build/pull/3213/commits/db0334c9afccc05a5a009a09a7dcf29b9bb0e037.